src/main/java/eiteam/esteemedinnovation/commons/init/ContentModuleHandler.java

```diff
@@ -101,6 +106,25 @@ public static void postInit() {
         }
     }
 
+    public static void registerBlocks(RegistryEvent.Register<Block> event) {
+        for (ContentModule module : modules) {
+            module.registerBlocks(event);
+        }
+    }
+
+    public static void registerItems(RegistryEvent.Register<Item> event) {
+        for (ContentModule module : modules) {
+            module.registerItems(event);
+        }
+    }
+
+    @SideOnly(Side.CLIENT)
+    public static void registerModels(ModelRegistryEvent event) {
```

Comment:
Would it be possible to generify these methods?

Reply:
Maybe, but forge fires `RegistryEvent`s for the different ["registries"](https://mcforge.readthedocs.io/en/latest/concepts/registries/#registering-things).